packages/analytics/analytics-metric-provider/src/components/MetricsProvider.cy.ts

```diff
@@ -454,11 +360,10 @@ describe('<AnalyticsMetricProvider />', () => {
     cy.get('.metricscard-trend-change > div').eq(2).should('have.text', '49.98%')
   })
 
-  it('displays single-entity metrics with no trend', () => {
+  it.skip('displays single-entity metrics with no trend', () => {
```

Comment:
Fine to drop this one, since we are now hardcoding `hasTrendAccess` to `true`?

Reply:
Yes, I think it's fine to drop this one.